### DISCUSSION TITLE
How to make multi bots from db

### DISCUSSION THREAD

**Discussion by RustamKakhramanov** [aiogram/aiogram#1442] [Q&A]
Hello! I dont understand how to mix (for example django) and aiogram. After send in my crud in admin panel i need that new telegram token set as new bot and use aiogram with other bots with single code without reboot aiogram

**Comment by Olegt0rr** [comment]
Check [this](https://github.com/aiogram/aiogram/blob/dev-3.x/examples/multibot.py) example and then implement it in your app
